compiler/build.gradle

```diff
@@ -11,5 +11,6 @@ dependencies {
 
     testImplementation "ch.qos.logback:logback-classic:$logbackVersion",
             "com.github.tomakehurst:wiremock-jre8:$wiremockVersion",
+            "org.testcontainers:testcontainers:$testcontainersVersion",
```

Comment:
why are you adding the testcontainers dependencies to all the projects?

Reply:
I removed it from the base build.gradle because it is not needed in all modules and moved it into the ones that actually require it. Just neuroticism at work.

Reply:
All fine!